.devcontainer/devcontainer.json

```diff
@@ -8,22 +8,15 @@
 		// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
 		"dockerfile": "./Dockerfile"
 	},
+	// To run docker in docker
+	"privileged": true,
 	"features": {
 		"ghcr.io/devcontainers/features/python:1": {}
 	},
 
-	// Install the local ddev cli at start time.
-	"postCreateCommand": "pip install -e /workspaces/integrations-core/ddev"
+	// Install the local ddev cli at start time and set the location of the integrations-core folder, start docker service
+	"postCreateCommand": "pipx install -e /workspaces/integrations-core/ddev && ddev config find && sed -i 's|core = \"~/dd/integrations-core\"|core = \"/workspaces/integrations-core\"|' /root/.local/share/dd-checks-dev/config.toml && service docker start",
```

Comment:
also you can set the path with the command `ddev config set core /workspaces/integrations-core`

Reply:
> what is the purpose of running service docker start I support we are using the docker socket from the host to run container. so the docker engine should already be up?

ddev e2e tests make use of docker-compose and currently expect to be run from the host with the container port being shared. For instance, the `ddev env start nginx py3.11-1.13` will fail with `ERROR tests/test_e2e.py::test_e2e - datadog_checks.dev.errors.RetryError: Endpoint: http://localhost:8080/nginx_status`. `localhost` represents the host machine, so it would fail from within the container. Thus it becomes necessary to use a new docker engine/socket instead of sharing the host machine.

> also you can set the path with the command ddev config set core /workspaces/integrations-core

Currently, `ddev config set` adds a new entry, irrelevantly on whether or not it is already set in the configuration file, thus we end up with a wrong configuration file and `ddev env` commands fail. Modifying in place the configuration ensures we have the desired result in the meantime.